tests/utils/utils.py

```diff
@@ -1,3 +1,35 @@
+from starkware.cairo.lang.tracer.profile import profile_from_tracer_data
+from starkware.cairo.lang.tracer.tracer_data import TracerData
+from starkware.starknet.business_logic.execution.execute_entry_point import (
+    ExecuteEntryPoint,
+)
+
+# Override _run() entrypoint for hooking and profiling cairo runner.
+old_run = ExecuteEntryPoint._run
+
+
+def override_run(*args, **kwargs):
```

Comment:
could be better to make it a decorator, like : 

```
def trace_run(run):
    def override_run(*args, **kwargs):
    # use the run param here and not `old_run`
          return

    return override_run

```

and in `conftest.py` we use a fixture with `autouse=True` to enable the behavior during the test suite

```
@pytest.fixutre(autouse=True)
def overide_run():
    ExecuteEntryPoint._run = trace_run(ExecuteEntryPoint._run)
    return
```